### PR TITLE
Added self test for data duplication

### PR DESCRIPTION
**Pull request by espegun**
Added self test in aggregate_dfs.py which raises an Exception if rows are duplicated (with regards to the columns "district", "bydelid" and "date").

### CODE REVIEW

File: common/aggregate_dfs.py [hunk 1]
```diff
@@ -43,6 +43,39 @@ def _check_data_consistency(df):
...
+    if len(check_for_these) == 0:
+        return None
+
+    duplicated_rows = df.duplicated(subset=check_for_these, keep=False)
+
+    if duplicated_rows.any():
+        print(df[duplicated_rows].sort_values(by=check_for_these))
+        raise ValueError(
+            'One or more rows have duplicated combinations of "district", "delbydelid" and "date".'
+        )
```
Comment (eide) [inline common/aggregate_dfs.py]: I think adding which rows are duplicated to the exception itself could be an improvement. But, I guess there are drawbacks. By using ValueError the message could become veeeery long, or you'd have to create a custom exception.

File: common/aggregate_dfs.py [hunk 2]
```diff
@@ -43,6 +43,39 @@ def _check_data_consistency(df):
...
+
+    Raises:
+        ValueError: If there are duplicated combinations of "district", "delbydelid" and "date".
+    """
+
+    CHECK_FOR_THESE = {"district", "delbydelid", "date"}
+    check_for_these = list(CHECK_FOR_THESE.intersection(set(df.columns)))
+
+    if len(check_for_these) == 0:
+        return None
```
[the remark this answers is not in the document]
Comment (espegun) [reply common/aggregate_dfs.py]: Done.

File: common/aggregate_dfs.py [hunk 3]
```diff
@@ -43,6 +43,39 @@ def _check_data_consistency(df):
...
+
+    duplicated_rows = df.duplicated(subset=check_for_these, keep=False)
+
+    if duplicated_rows.any():
+        print(df[duplicated_rows].sort_values(by=check_for_these))
+        raise ValueError(
+            'One or more rows have duplicated combinations of "district", "delbydelid" and "date".'
+        )
+
+    return None
```
[the remark this answers is not in the document]
Comment (espegun) [reply common/aggregate_dfs.py]: Done.

File: common/aggregate_dfs.py (hunk 1 above)
Comment (espegun) [reply common/aggregate_dfs.py]: Kept the print statement, but only df.head(), i.e. the top five rows of duplicated rows.
This should balance the need to point out the cause of the error, while avoiding very long prints.

Comment (espegun) [reply common/aggregate_dfs.py]: df.head() included in ValueError string, nicely formatted.
Also minor changes in section when the file is run separately, only used for debugging and demonstration.

**Pull request merged**